packages/spaces/src/lib/spaces-types.tsx

```diff
@@ -26,38 +32,30 @@ export type Space = {
   /** A list of payerIds for the configuration or content. */
   payerIDs?: string[];
   /** The images associated with the configuration. */
-  images?: NameValuePair[];
+  images?: SpacesImages;
+  //** Metadata for a configuration */
+  meta?: { [key: string]: any };
```

Comment:
For this and the other types like it I think it's preferred that we use `unknown` instead of `any` and use a `Record`. I believe it should help with having people cast the type if needed. There's also an extra `/` in the comment.

`meta?: Record<string, unknown>`